src/services/getLocalData.tsx

```diff
@@ -1,13 +1,25 @@
-export function getLogStatus(): any {
+export function getLogStatus(): boolean {
     const val=localStorage.getItem("LogStatus")?localStorage.getItem("LogStatus"):"false";
```

Comment:
I think the code in this function can be reduced with return localStorage.getItem("LogStatus")?Boolean(localStorage.getItem("LogStatus")):false;

Reply:
Dividing the code may be more readable. For example:
`
const logStatus = localStorage.getItem("LogStatus")
const val = logStatus ? logStatus : "false
 `

Reply:
Yes Aayush. We can return the bool of val. if(val=="true"){ return true } else{ return false } this part seemed redundant in the above code and could be reduced.

Reply:
Yes Aayush. We can return the bool of val. `if(val=="true"){ return true } else{ return false }` this part seemed redundant in the above code and could be reduced.

Reply:
Changed it.

Reply:
> 

It isn't working. console.log(Boolean("false")) is true.

Reply:
Boolean of some string will return true. what I meant was check your localStorage if there is some string, Boolean of that string will give you true. if there is nothing in your localStorage return false. In the image below, there is no ABC in localStorage, so it returned false. in the second, localStorage was set so it returned true.
![image](https://user-images.githubusercontent.com/60066504/182298280-4e191136-6a4a-4270-8a02-5310efd2a1a8.png)

Reply:
If you are setting false and true as a string in your localStorage. While accessing those values make sure to parse those string.
![image](https://user-images.githubusercontent.com/60066504/182303125-cb45c6d4-1d4a-4e1b-a491-da83b014a84c.png)

Reply:
Ok. I will try.